src/aosm/azext_aosm/common/constants.py

```diff
@@ -59,12 +60,14 @@ class ManifestsExist(str, Enum):
 NSD_BASE_TEMPLATE_FILENAME = "nsdbase.bicep"
 NSD_TEMPLATE_FOLDER_NAME = "nsd"
 NSD_DEFINITION_FOLDER_NAME = "nsdDefinition"
+NSD_NF_TEMPLATE_FILENAME = "nf_template.bicep.j2"
 
 VNF_OUTPUT_FOLDER_FILENAME = "vnf-cli-output"
 VNF_INPUT_FILENAME = "vnf-input.jsonc"
 VNF_DEFINITION_TEMPLATE_FILENAME = "vnfdefinition.bicep.j2"
 VNF_MANIFEST_TEMPLATE_FILENAME = "vnfartifactmanifest.bicep.j2"
-VNF_BASE_TEMPLATE_FILENAME = "vnfbase.bicep"
+VNF_CORE_BASE_TEMPLATE_FILENAME = "vnfbase.bicep"
+VNF_NEXUS_BASE_TEMPLATE_FILENAME = "vnfnexusbase.bicep"
```

Comment:
This list feels like it's very inefficient. Not saying we do it now, but we should have in mind if we can cut the list down a bit.